clientv3/snapshot/v3_snapshot.go

```diff
@@ -267,8 +267,8 @@ func (s *v3Manager) Restore(cfg RestoreConfig) error {
 	if dataDir == "" {
 		dataDir = cfg.Name + ".etcd"
 	}
-	if fileutil.Exist(dataDir) {
-		return fmt.Errorf("data-dir %q exists", dataDir)
+	if fileutil.Exist(dataDir) && !fileutil.Empty(dataDir) {
+		return fmt.Errorf("data-dir %q not empty", dataDir)
```

Comment:
@zhanw15 thanks for the PR! Please wait on this because a separate discussion is going on in handling existing files. I will be updating on it to you here. Also, if we decide to go further, a small suggestion on modifying the error message to something like `data-dir %q not empty or could not be read` considering any read error in the `Empty` function? Thanks again!

Reply:
Yes, I think you are right, now I change the logs info.